src/lazy/deferred.ts

```diff
@@ -26,7 +27,21 @@ export const createDeferred = <C extends ComponentType<any>>(
       };
     }),
     result: undefined,
-    start: () => loader().then(resolve),
+    preload: () => {
+      loader().then((m: any) => {
```

Comment:
We could check for `deferred.result` here too, and if already there do nothing

Reply:
Sure